### DISCUSSION TITLE
Save [search query + difficulty range + sort order + collection] in filter presets

### DISCUSSION THREAD

**Discussion by SamedhG** [ppy/osu#30403] [Ideas]
When I start osu I often end up filtering down my beatmaps  by typing in `ar>8 cs<4` or something and selecting a difficulty range. I have a handful of different search queries that I use repeatedly. Would be good if I could save these as presets and select them from a dropdown. 

I'm happy to implement this but would need a little time to familiarize myself with the codebase

I this something people would be interested in?

**Comment by Gabixel** [comment]
The closest idea similar to this is probably the suggested ["Dynamic Collections"](https://github.com/ppy/osu/discussions/13010#discussioncomment-791245), which is something that seems to already be on the devs to-do list. (Not sure about the sort order.)

**Reply by SamedhG** [reply]
Oh that's cool. Kinda covers what my feature would do.
